pkg/contexts/ctxlog/ctxlog.go

```diff
@@ -21,6 +21,11 @@ func FromContext(ctx context.Context) log.Logger {
 		return log.NewNopLogger()
 	}
 	span := trace.FromContext(ctx).SpanContext()
+
+	if span.TraceID.String() == "00000000000000000000000000000000" {
```

Comment:
perhaps a comment explaining this? Is the exact number of zero's significant?

Reply:
Refactored. Probably more self describing now